spec/workers/atmosphere/vm_load_monitoring_worker_spec.rb

```diff
@@ -6,6 +6,14 @@
     Fog.mock!
   }
 
+  let!(:vm1) { create(:virtual_machine, ip: '10.100.0.1', monitoring_id: 1,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.